code/modules/food_and_drinks/restaurant/customers/_customer.dm

```diff
@@ -43,9 +43,24 @@
 	. = ..()
 	name_prefixes = world.file2list(prefix_file)
 
+/// Can this customer be chosen for this venue?
+/datum/customer_data/proc/can_use(datum/venue/venue)
+	return TRUE
+
+/// Called when the venue chooses this customer
+/datum/customer_data/proc/chosen(datum/venue/venue)
+
+/// Gets the order of this customer.
+/// In most cases, you shouldn't override this, and should just modify orderable_objects.
```

Comment:
Rather than saying that you shouldn't override this, explain when you might want to override it, which is generally when you want something thats based on the customers environment.